common.sh

```diff
@@ -210,7 +210,8 @@ export KNI_INSTALL_FROM_GIT=${KNI_INSTALL_FROM_GIT:-}
 #
 # if we provide OPENSHIFT_RELEASE_IMAGE, do not curl. This is needed for offline installs
 if [ -z "${OPENSHIFT_RELEASE_IMAGE:-}" ]; then
-  LATEST_CI_IMAGE=$(curl https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/4.6.0-0.ci/latest | grep -o 'registry.svc.ci.openshift.org[^"]\+')
+  OPENSHIFT_RELEASE_STREAM=${OPENSHIFT_RELEASE_STREAM:-4.6}
+  LATEST_CI_IMAGE=$(curl https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/${OPENSHIFT_RELEASE_STREAM}.0-0.ci/latest | grep -o 'registry.svc.ci.openshift.org[^"]\+')
```

Comment:
It would be good to add "-L" option to the curl, to follow redirects. The change is not related to this PULL but worth noting it.